src/main/kotlin/io/appwrite/Client.kt

```diff
@@ -62,11 +62,11 @@ class Client @JvmOverloads constructor(
     init {
         headers = mutableMapOf(
             "content-type" to "application/json",
-            "user-agent" to "AppwriteKotlinSDK/4.0.0 ${System.getProperty("http.agent")}",
+            "user-agent" to "AppwriteKotlinSDK/4.1.0 ${System.getProperty("http.agent")}",
             "x-sdk-name" to "Kotlin",
             "x-sdk-platform" to "server",
             "x-sdk-language" to "kotlin",
-            "x-sdk-version" to "4.0.0",            "x-appwrite-response-format" to "1.4.0"
+            "x-sdk-version" to "4.1.0",            "x-appwrite-response-format" to "1.4.0"
```

Comment:
```suggestion
            "x-sdk-version" to "4.1.0",
            "x-appwrite-response-format" to "1.4.0"
```

Reply:
Not caused by this release; will address in the next